### PR TITLE
Update changelog - it should say something about new Python verison requirements

### PR DESCRIPTION
**Pull request by ekoval**
Related to: #175

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ Change Log
 -----
 
 * Make `hooks.django.DjangoApp.manage_py` method return command output.
+  (this change is incompatible with Python<=2.6)
```
Comment (stefanor) [inline CHANGELOG.md]: I'd go further than that and say "Dropped support for Python 2.6".

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ Change Log
 -----
 
 * Make `hooks.django.DjangoApp.manage_py` method return command output.
+  (Dropped support for Python 2.6)
```
Comment (stefanor) [inline CHANGELOG.md]: No, I mean, make it a top level item. This is probably the most important thing in the changelog.

That there was a bug fix in manage_py is relatively unimportant to consumers of the changelog (I mean, what the details were, really all they want to know is it didn't work in the previous release, it does now). But removing support for a Python version is.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by ekoval** [comment]
:bell:  @stefanor

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 Change Log
 ==========
 
+NOTE: Versions >= 0.7.3 require Python >= 2.7.
```
Comment (stefanor) [inline CHANGELOG.md]: Surely that belongs under the 0.7.3 section?

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Pull request merged**